CONTRIBUTING.md

```diff
@@ -233,3 +233,16 @@ In a rare case you may want to release only one specific package. To do so, foll
 4. Create a draft PR (not intended to be merged) to the clay repo; the sole purpose of this is to see CI green one last time before pushing the tag.
 5. Once CI is green, run `git push $REMOTE master --follow-tags`
 6. Run `yarn publish` to publish module to the npm registry.
+
+## Updating release in [liferay-portal](https://github.com/liferay/liferay-portal)
+
+1. Navigate to `./modules/apps/frontend-taglib/frontend-taglib-clay`
+2. Run `ncu '/@clayui/' -u` ([npm-check-updates](https://www.npmjs.com/package/npm-check-updates))
+3. Navigate to `./portal-impl` and run `ant format-source-all`
+   3a. Often this will also update other files other than `package.json`s and the `yarn.lock`. Disregard changes to files other than those.
```

Comment:
Formatting is off here when rendered as Markdown. I don't think you can do the `3a.` thing:

![Screenshot 2020-07-14 -120128-mUZSsKMh@2x](https://user-images.githubusercontent.com/7074/87412830-c9070400-c5c9-11ea-8207-837d302f7383.png)

Ironically, it *does* work in GitHub comments 😂:

---

1. Navigate to `./modules/apps/frontend-taglib/frontend-taglib-clay`
2. Run `ncu '/@clayui/' -u` ([npm-check-updates](https://www.npmjs.com/package/npm-check-updates))
3. Navigate to `./portal-impl` and run `ant format-source-all`
   3a. Often this will also update other files other than `package.json`s and the `yarn.lock`. Disregard changes to files other than those.
4. Run `yarn` from `./modules`

---

I think nested *numbers* works though:

```markdown
1.  Navigate to `./modules/apps/frontend-taglib/frontend-taglib-clay`
2.  Run `ncu '/@clayui/' -u` ([npm-check-updates](https://www.npmjs.com/package/npm-check-updates))
3.  Navigate to `./portal-impl` and run `ant format-source-all`
    1. Often this will also update other files other than `package.json`s and the `yarn.lock`. Disregard changes to files other than those.
4.  Run `yarn` from `./modules`
```

Rendered as Markdown doc:

![Screenshot 2020-07-14 -120425-Wz2KVmiK@2x](https://user-images.githubusercontent.com/7074/87413146-33b83f80-c5ca-11ea-93af-049046fd59ad.png)

Rendered as comment:

---

1.  Navigate to `./modules/apps/frontend-taglib/frontend-taglib-clay`
2.  Run `ncu '/@clayui/' -u` ([npm-check-updates](https://www.npmjs.com/package/npm-check-updates))
3.  Navigate to `./portal-impl` and run `ant format-source-all`
    1. Often this will also update other files other than `package.json`s and the `yarn.lock`. Disregard changes to files other than those.
4.  Run `yarn` from `./modules`